docs/changes.rst

```diff
@@ -6,6 +6,11 @@ Pending
 
 * Added Django 5.2 to the tox matrix.
 
+5.0.1 (2025-02-15)
+------------------
+
+* Updated package metadata to include well-known labels.
```

Comment:
```suggestion
* Updated package metadata to include well-known labels.
```
We ask that people put the change log lines for their work under the Pending header. When we cut a release, we'll rename that header with the version and date.